src/test/java/com/synopsys/integration/jenkins/extensions/JenkinsIntLoggerTest.java

```diff
@@ -102,14 +102,16 @@ public void testJenkinsIntLoggerTaskListener() {
     @ParameterizedTest
     @MethodSource("txtTestData")
     public void testJenkinsIntLogger_alwaysLog(String txt, Throwable e, LogLevel logLevel, String expected) {
-        if (e == null) {
-            JenkinsIntLogger jenkinsIntLogger = new JenkinsIntLogger(null);
-            jenkinsIntLogger.setLogLevel(logLevel);
-            jenkinsIntLogger.alwaysLog(txt);
-            assertNull(jenkinsIntLogger.getTaskListener());
-            assertEquals(logLevel, jenkinsIntLogger.getLogLevel());
-            assertEquals(expected, byteArrayOutputStream.toString());
+        if (e != null) {
+            throw new IllegalArgumentException("Test requires that e is null");
```

Comment:
I would consider doing Assertions.assertNull here with a helpful message instead of doing a null check and throwing an IllegalArgumentException